apps.json

```diff
@@ -1424,6 +1424,16 @@
         "state": "working",
         "url": "https://github.com/yunohost-apps/humhub_ynh"
     },
+    "icecoder": {
+        "branch": "master",
+        "category": "dev",
+        "revision": "HEAD",
+        "state": "inprogress",
+        "subtags": [
+            "programming"
+        ],
+        "url": "https://github.com/YunoHost-Apps/icecoder_ynh",
```

Comment:
There is an extra comma right before the closing bracket.

Reply:
Ok!